ztp/source-crs/SriovOperatorConfig.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
 spec:
   configDaemonNodeSelector:
     "node-role.kubernetes.io/$mcp": ""
-  disableDrain: false
```

Comment:
Did you confirm that for both SNO and multi-node clusters the Operator will set this to the correct value if not supplied by the user?

Reply:
> Did you confirm that for both SNO and multi-node clusters the Operator will set this to the correct value if not supplied by the user?

Yes, have confirmed. Removing this line keeps the exact the same setting as today. For SNO, disableDrain is auto-set to true. For muli-node clusters, this field will be false(by default)